web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java

```diff
@@ -46,6 +47,9 @@
   @JsonProperty("neighbor_table")
   private List<NeighborEntry> neighborTable;
 
+  @JsonProperty("routerinfo")
+  private List<InternalRouterInfo> routerTable;
```

Comment:
Suggest to rename the variable name to routerInfo.

Reply:
fixed in a90538a

Reply:
@haboy52581 the variable name itself remains unchanged.